src/generators/minecraft-axolotl-character/MinecraftAxolotlCharacterGenerator.res

```diff
@@ -0,0 +1,1014 @@
+let requireImage = id => Generator.requireImage("./images/" ++ id ++ ".png")
+let requireTexture = id => Generator.requireImage("./textures/" ++ id ++ ".png")
+
+let id = "minecraft-axolotl-character"
+
+let name = "Minecraft Axolotl Character"
+
+let thumbnail: Generator.thumnbnailDef = {
+  url: Generator.requireImage("./thumbnail/v2thumbnail_256.jpeg"),
+}
+
+let imageIds = ["Background", "Folds", "Labels"]
+let toImageDef = (id): Generator.imageDef => {id: id, url: requireImage(id)}
+let images: array<Generator.imageDef> = imageIds->Js.Array2.map(toImageDef)
+
+let textures: array<Generator.textureDef> = [
+  {
+    id: "Skin",
+    url: requireTexture("Steve"),
+    standardWidth: 64,
+    standardHeight: 64,
+  },
+  {
+    id: "Blue",
+    url: requireTexture("axolotl_blue"),
+    standardWidth: 64,
+    standardHeight: 64,
+  },
+  {
+    id: "Cyan",
+    url: requireTexture("axolotl_cyan"),
+    standardWidth: 64,
+    standardHeight: 64,
+  },
+  {
+    id: "Pink",
+    url: requireTexture("axolotl_lucy"),
+    standardWidth: 64,
+    standardHeight: 64,
+  },
+  {
+    id: "Gold",
+    url: requireTexture("axolotl_gold"),
+    standardWidth: 64,
+    standardHeight: 64,
+  },
+  {
+    id: "Brown",
+    url: requireTexture("axolotl_wild"),
+    standardWidth: 64,
+    standardHeight: 64,
+  },
+]
+
+let steve = TextureMap.MinecraftCharacterLegacy.steve
+let alex = TextureMap.MinecraftCharacterLegacy.alex
```

Comment:
I've just pushed up a change to `main` which updates the `TextureMap` file.

Can you [get that latest change into your branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork)?

Then you can updated these two lines:

```
let steve = TextureMap.MinecraftCharacter.steve
let alex = TextureMap.MinecraftCharacter.alex
```

We will use these below.